### PR TITLE
Fix headings to wrap optimally before solver

### LINKED ISSUE #51: Solver does not wrap -pi to pi

### PR DESCRIPTION
**Pull request by shueja**
Closes #51

### CODE REVIEW

File: src/document/HolonomicPathStore.ts [hunk 1]
```diff
@@ -329,6 +330,33 @@ export const HolonomicPathStore = types
...
+              let prevHeadingMod = angleModulus(prevHeading);
+              let heading = pts[i].heading;
+              let headingMod = angleModulus(heading);
+              if (prevHeadingMod < 0 && headingMod > prevHeadingMod + Math.PI) {
+                // negative rollunder
+                fullRots--;
+              } else if (
+                prevHeadingMod > 0 &&
+                headingMod < prevHeadingMod - Math.PI
+              ) {
```
Comment (Awesomeplayer165) [inline src/document/HolonomicPathStore.ts:350]: Consider making the code more consistent

**Review by Awesomeplayer165 — CHANGES_REQUESTED** [review]
Formatting things but general functionality lgtm

**Comment by shueja** [comment]
> Formatting things but general functionality lgtm

I tried to fix this and the formatter reverted it. I'm not going to try to fight it to make the line breaks consistent

**Comment by Awesomeplayer165** [comment]
no problem

**Review by Awesomeplayer165 — APPROVED** [review]

**Pull request merged**